### PR TITLE
Fix a few bugs for Stats

### PR DESCRIPTION
**Pull request by lzchen**
rpid -> rpId
Remove single "/" in rpId for vm
Change disableAll env to check for explicit 'True" string

Also fixed some constructor tests.

### CODE REVIEW

File: contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def __init__(self, **options):
         self._worker = Worker(self._queue, self)
         self._worker.start()
         # start statsbeat on exporter instantiation
-        if not os.environ.get("APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL"):
+        if os.environ.get("APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL") != "True":
```
Comment (hectorhdzg) [inline contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py:67]: APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL=true will not work in this case, we usually just check for the env variable to exist and value is not that important, if you want to check for value ensure lower case and extra spaces doesn't mess with the check

Comment (lzchen) [reply contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py:67]: If it's normal practice to check if it exists then that is fine with me. Will leave it as a `None` check then.

**Review by hectorhdzg — APPROVED** [review]
LGTM

File: contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py (hunk 1 above)
Comment (heyams) [reply contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py]: value matters though.. 

convert string to boolean and then check it.  Boolean.parseBoolean(value) (java way) handle case for you.

File: contrib/opencensus-ext-azure/opencensus/ext/azure/trace_exporter/__init__.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def __init__(self, **options):
         super(AzureExporter, self).__init__(**options)
         atexit.register(self._stop, self.options.grace_period)
         # start statsbeat on exporter instantiation
-        if not os.environ.get("APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL"):
+        if os.environ.get("APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL") != "True":
```
Comment (heyams) [inline contrib/opencensus-ext-azure/opencensus/ext/azure/trace_exporter/__init__.py]: check boolean value to support case sensitivity.

File: contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py (hunk 1 above)
Comment (lzchen) [reply contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py]: I'm fine either way, just trying to be consistent with node js sdk. @hectorhdzg  thoughts?

Comment (hectorhdzg) [reply contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py]: Like I mentioned node.js SDK does this kind of only checking for env to exist and not checking the value, we also mention this in docs when apply, is up to you how to handle it, casting works as well

Comment (lzchen) [reply contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py]: Will leave it as None.

**Pull request merged**